pkg/api/snapshots.go

```diff
@@ -6,15 +6,16 @@ import (
 )
 
 type SnapshotResponse struct {
-	UUID           string           `json:"uuid"`
-	CreatedAt      time.Time        `json:"created_at"`      // Datetime the snapshot was created
-	RepositoryPath string           `json:"repository_path"` // Path to repository snapshot contents
-	ContentCounts  map[string]int64 `json:"content_counts"`  // Count of each content type
-	AddedCounts    map[string]int64 `json:"added_counts"`    // Count of each content type
-	RemovedCounts  map[string]int64 `json:"removed_counts"`  // Count of each content type
-	URL            string           `json:"url"`             // URL to the snapshot's content
-	RepositoryName string           `json:"repository_name"` // Name of repository the snapshot belongs to
-	RepositoryUUID string           `json:"repository_uuid"` // UUID of the repository the snapshot belongs to
+	UUID            string           `json:"uuid"`
+	CreatedAt       time.Time        `json:"created_at"`                            // Datetime the snapshot was created
+	RepositoryPath  string           `json:"repository_path"`                       // Path to repository snapshot contents
+	ContentCounts   map[string]int64 `json:"content_counts"`                        // Count of each content type
+	AddedCounts     map[string]int64 `json:"added_counts"`                          // Count of each content type
+	RemovedCounts   map[string]int64 `json:"removed_counts"`                        // Count of each content type
+	URL             string           `json:"url"`                                   // URL to the snapshot's content
+	RepositoryName  string           `json:"repository_name"`                       // Name of repository the snapshot belongs to
+	RepositoryUUID  string           `json:"repository_uuid"`                       // UUID of the repository the snapshot belongs to
+	PublicationHref string           `json:"publication_href" swaggerignore:"true"` // Publication href of the snapshot in pulp
```

Comment:
if swaggerignore is true, does that require an api doc update? looking at the PR check failure: `failed on setup with "iqe_content_sources_api.exceptions.ApiAttributeError: ApiSnapshotResponse has no attribute 'publication_href' at ['received_data']['data'][0]['last_snapshot']['publication_href']"`

Reply:
oh, I also meant to do `json:"-"`. That will allow it to be used in the struct, but not in API response